sdk/helper/docker/testhelpers.go

```diff
@@ -207,7 +207,7 @@ var _ io.Writer = &LogConsumerWriter{}
 func (d *Runner) StartNewService(ctx context.Context, addSuffix, forceLocalAddr bool, connect ServiceAdapter) (*Service, string, error) {
 	if d.RunOptions.PreDelete {
 		name := d.RunOptions.ContainerName
-		matches, err := d.DockerAPI.ContainerList(ctx, types.ContainerListOptions{
+		matches, err := d.DockerAPI.ContainerList(ctx, container.ListOptions{
```

Comment:
These aren't technically required, and I can revert if requested, but the previous form is deprecated and is removed in the next release.